tests/test_text_cross_encoder.py

```diff
@@ -80,3 +93,30 @@ def test_lazy_load(model_name):
 
     if is_ci:
         delete_model_cache(model.model._model_dir)
+
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        model_desc["model"]
+        for model_desc in TextCrossEncoder.list_supported_models()
+        if model_desc["size_in_GB"] < 1 and model_desc["model"] in CANONICAL_SCORE_VALUES.keys()
+    ],
```

Comment:
```suggestion
    [
        "Xenova/ms-marco-MiniLM-L-6-v2",
        "BAAI/bge-reranker-base",
        "jinaai/jina-reranker-v1-tiny-en"
    ],
```

Reply:
I think that we can create a set of model names, e.g. one of each provider (Xenova/BAAI/jinaai) and reuse it in tests like test_batch_embedding, test_parallel_processing, etc

Reply:
What's the goal?

Reply:
Speed up CI?

Reply:
Yes, to speed up CI and tests in general
We run all (almost) models in  `test_rerank` test, I think there is no need to run more complicated tests with all of the models and only one per family of models is quite enough

Reply:
Ok, sure